pages/osx/fileicon.md

```diff
@@ -0,0 +1,20 @@
+# fileicon
+
+> a macOS CLI for managing custom icons for files and folders, as a programmatic alternative to interactively using Finder.
+> More information: <https://github.com/mklement0/fileicon>.
+
+- Assign custom icon derived from image file 'img.png' to file 'foo':
+
+`fileicon set foo img.png`
+
+- Remove previously assigned custom icon from folder 'foodir':
+
+`fileicon rm foodir`
```

Comment:
```suggestion
- Remove a custom icon from folder specific file or directory:

`fileicon rm {{path/to/file_or_directory}}`
```

Reply:
> folder specific file or directory 

this doesn't make sense, grammatically.

Change it to something like

> from a specific file or directory

Reply:
@marchersimon I think you meant to replace `folder` with `a`, so I fixed that ;)